### ISSUE TITLE
Mobile app: don't open after hoarding

### ISSUE THREAD

**Issue by DominikBritz** [hoarder-app/hoarder#237]
When I hoard something via the Android share menu, I get the black "Hoarded!" screen and then the mobile app opens.

Can we have an option to not show the app? I mean my point of hoarding is to read it later - seeing the app is unnecessary.

Thanks for this awesome project btw., ditched Pocket for Hoarder!

**Comment by MikeKMiller** [comment]
That would be nice....

**Comment by MohamedBassem** [comment]
Yeah ... I know. Unfortunately, it's currently a limitation in the library we're using for the share sheet. I'll try to open an issue to that library and see what would it take to allow us to support that.

**Comment by lucius346346** [comment]
Agreed. It should probably display a toast message or a toolbar, like Pocket has.

**Comment by zjw2018SS** [comment]
Still very relevant in 2026 — sharing from the Android share sheet still launches the full Karakeep app.

I took another look at the current implementation: Karakeep still uses `expo-share-intent`, and the library's design is explicitly to redirect shared content to the main application for processing. As of v8.0.1, there is no opt-in capture mode that avoids launching the main app.

It seems there are two possible paths:

1. upstream support in `expo-share-intent` for a non-main-app sharing flow, or
2. a Karakeep-specific native share target / share extension, with `expo-share-intent` disabled for that platform — the library itself offers `disableAndroid` / `disableIOS` options for custom platform implementations.

The second option may be worth considering, since an upstream change would go against the library's core "redirect to main app" design.

The ideal UX would be:

Share → Karakeep → save in place → brief confirmation → remain in the originating app.

Related: #3074 proposes an Android Quick Settings tile for the same general goal of capture without context switching.
